Orange/canvas/report/report.py

```diff
@@ -162,8 +162,7 @@ def report_plot(self, name=None, plot=None):
             self.report_html += get_html_img(plot.scene())
 
     # noinspection PyBroadException
-    def report_table(self, name, table=None, header_rows=0, header_columns=0,
-                     num_format=None):
+    def report_table(self, name, table=None, header_rows=0, header_columns=0, tot_rows = 0, num_format=None):
```

Comment:
1. [PEP-8](https://www.python.org/dev/peps/pep-0008/) states the correct way is to specify function parameter defaults as `key=value`, that is, without whitespace around the equals operator.
2. `tot_rows` is not a good identifier name. What does it stand for?